### PR TITLE
add option to edit VM

### PR DESCRIPTION
**Pull request by mkrajnak**

### CODE REVIEW

**Comment by mkrajnak** [comment]
@mareklibra @matobet @jniederm @jelkosz please review

**Review by matobet — APPROVED** [review]

**Review by mareklibra — APPROVED** [review]

File: src/actions/vm.js [hunk 1]
```diff
@@ -76,6 +76,15 @@ export function suspendVm ({ vmId }) {
   }
 }
 
+export function editVm ({ vm }) {
```
Comment (jniederm) [inline src/actions/vm.js:79]: Please rename it to `vmId`. Javascript is not a typed, it took me a while to realize it's not an object.

**Review by jniederm — COMMENTED** [review]
Corresponding redurer is in web-ui, right?

File: src/actions/vm.js (hunk 1 above)
Comment (mkrajnak) [reply src/actions/vm.js:79]: @jniederm well it is not vmId its whole vm, editVm dialog is filled with data from this payload

**Review by jniederm — APPROVED** [review]

**Comment by matobet** [comment]
Looks good but merging this should wait until the vm-dialog is complete on the ovirt-web-ui https://github.com/oVirt/ovirt-web-ui/pull/98 side

**Comment by mareklibra** [comment]
Rebase is needed because of https://github.com/oVirt/ovirt-ui-components/pull/35.
Please consider moving the components to ovirt-web-ui.

**Comment by mkrajnak** [comment]
Since components affected by this patch have moved to oVirt-web-ui I am moving the patch there to my pending pull request and I am closing this.

**Pull request closed without merging**